### PR TITLE
ZTE-SH-CN-images.md

### PR DESCRIPTION
**Pull request by zhangmingld**
translation to the docs/concept/containers/images.md

> ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> For 1.8 Features: set Milestone to `1.8` and Base Branch to `release-1.8`
> ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>
> NOTE: Please check the “Allow edits from maintainers” box (see image below) to
> [allow reviewers to fix problems](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/) on your patch and speed up the review process.
>
> Please delete this note before submitting the pull request.

Allow edits from maintainers checkbox

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/5453)

### CODE REVIEW

**Comment by chenopis** [comment]
cc @zhangxiaoyu-zidif @markthink

File: cn/docs/concepts/containers/images.md [hunk 1]
```diff
@@ -0,0 +1,299 @@
...
+  - pod上指定 ImagePullSecrets
+    - 只有提供了密钥的pod才能接入私有仓库
+下面将详细描述每一项
+
+
+### 使用 Google Container Registry
+Kuberetes运行在Google Compute Engine (GCE)时原生支持[Google ContainerRegistry (GCR)]
+(https://cloud.google.com/tools/container-registry/)。如果kubernetes集群运行在GCE
+或者Google Container Engine (GKE)上，使用镜像全名(e.g. gcr.io/my_project/image:tag)
+即可。
```
Comment (zhangxiaoyu-zidif) [inline cn/docs/concepts/containers/images.md]: “即可” 不需要换行。

File: cn/docs/concepts/containers/images.md [hunk 2]
```diff
@@ -0,0 +1,299 @@
...
+
+集群中的所有pod都会有读取这个仓库中镜像的权限。
+
+Kubelet将使用实例的Google service account向GCR认证。实例的service account拥有
+`https://www.googleapis.com/auth/devstorage.read_only`，所以它可以从项目的GCR拉取，但不能
```
Comment (zhangxiaoyu-zidif) [inline cn/docs/concepts/containers/images.md]: 无需换行。

File: cn/docs/concepts/containers/images.md [hunk 3]
```diff
@@ -0,0 +1,299 @@
...
+- 验证是否满足以上要求
+- 获取工作站的$REGION (例如 `us-west-2`)凭证，使用凭证SSH到主机手动运行docker，检查是否运行
+- 验证kublet是否使用参数`--cloud-provider=aws`运行
+- 检查kubelet日志(例如 `journalctl -u kubelet`)，是否有类似的行
+  - `plugins.go:56] Registering credential provider: aws-ecr-key`
+  - `provider.go:91] Refreshing cache for provider: *aws_credentials.ecrProvider`
+
+### 使用 Azure Container Registry (ACR)
+当使用[Azure Container Registry](https://azure.microsoft.com/en-us/services/container-registry/)时，
+可以使用admin user或者service principal认证。
```
Comment (zhangxiaoyu-zidif) [inline cn/docs/concepts/containers/images.md]: 换行问题。

**Pull request merged**